ci-operator/step-registry/multiarch/nodes-readiness/multiarch-nodes-readiness-ref.yaml

```diff
@@ -0,0 +1,43 @@
+ref:
+  as: multiarch-nodes-readiness
+  from: upi-installer
```

Comment:
Sounds like the step did not use other tools beside oc, so I guess we can use `from: cli` here.

Reply:
i'm using yq now to resolve another issue. As it is included in upi-installer, i reverted to it.